backend/src/main/java/de/neuefische/backend/service/TravelService.java

```diff
@@ -0,0 +1,33 @@
+package de.neuefische.backend.service;
+import de.neuefische.backend.model.QuestionsCatalog;
+import de.neuefische.backend.repo.TravelRepo;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+
+@Service
+public class TravelService {
+
+    private final TravelRepo repo;
+
+    @Autowired
+    public TravelService(TravelRepo repo) {
+        this.repo = repo;
+    }
+
+    public List<QuestionsCatalog> getAllAnswers() {
+        return repo.findAll(); //konnte ich anstatt findall() > getAllAnswers schreiben?
+    }
+
+    public QuestionsCatalog postAnswers(QuestionsCatalog questionsCatalog) {
+        return repo.save(questionsCatalog);
+    }
+
+/*    public QuestionsCatalog postAnswers(QuestionsCatalog questionsCatalog) {
+        questionsCatalog.setId(idService.generateId());
+
+        return repo.save(movie);
+    }*/
```

Comment:
Wolltest du hier mit einer RandomId in die Datenbank einspeisen? 
Wenn ja, dann probier das mal so.
Ansonsten am Ende des Projekts die comments löschen.
```suggestion
   
     public QuestionsCatalog postAnswerst(QuestionsCatalog questionsCatalog) {
        questionsCatalog.setId(idService.generateId());
        
        return repo.save(questionsCatalog);

    }
```

Reply:
Genau, du hast recht
Ich wollte dran weiter arbeiten. 
Bei Pull Request habe ich aber vergessen, um das zu löschen